easybuild/easyblocks/l/lammps.py

```diff
@@ -469,7 +470,7 @@ def install_step(self):
 
             mkdir(site_packages, parents=True)
 
-            self.lammpsdir = os.path.join(self.builddir, '%s-stable_%s' % (self.name.lower(), self.version))
+            self.lammpsdir = os.path.join(self.builddir, '%s-*_%s' % (self.name.lower(), self.version))
```

Comment:
This seems to be the only real necessary change, the PR title should reflect that?